lib/pages/connect/connect_page_mixin.dart

```diff
@@ -153,4 +203,23 @@ mixin ConnectPageMixin {
     }
     return list;
   }
+
+  void handleTokenFromRegistrationSite({
+    required MatrixState matrix,
+    required String uri,
+  }) async {
+    final token = Uri.parse(uri).queryParameters['loginToken'];
+    Logs().d(
+      "ConnectPageMixin: handleTokenFromRegistrationSite: token: $token",
+    );
+    if (token?.isEmpty ?? false) return;
```

Comment:
The condition is not correct, if token is null => false => continue execution but not return ?